frontend/common/src/components/SideMenu/SideMenuContentWrapper.tsx

```diff
@@ -0,0 +1,7 @@
+import React from "react";
+
+const SideMenuContentWrapper: React.FC = ({ children }) => (
```

Comment:
Could we call it something more descriptive, so it's more legible? You mentioned it's for wide screens, and it'd be great to know that without diving down a few layers into css.

Also, any reason the SideMenu wouldn't want this? Why it's own component?

Reply:
Note, the naming is lack of creativity on my end and any suggestions would be helpful to better describe the components purpose.

Reply:
That's helpful! I didn't realize the relation from the name, but I can see now why it makes sense :)

Reply:
This is resolved unless you have any ideas to make confusion like mine go away haha

Reply:
This is helpful! Going to resolve